roles/DCOS.agent_windows/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# AWS S3 bucket with additional files:
+# https://dcos-win.s3.amazonaws.com/vc_redist.x64.exe
+# https://dcos-win.s3.amazonaws.com/1.13/vmesos-binaries.zip
+bucket_name: "dcos-win.s3.amazonaws.com"
```

Comment:
this should better reference to URLs so people can mirror and cache. Also for the defaults and release it should point to https://downloads.mesosphere.com

Reply:
Does this also relates to the alpha version? For Beta I agree, but shouldn't be s3 good enough for now?